test/cpp/api/functional.cpp

```diff
@@ -272,6 +272,13 @@ TEST_F(FunctionalTest, Hardshrink) {
   }
 }
 
+TEST_F(FunctionalTest, Arange) {
```

Comment:
This test should live in `test/cpp/api/tensor.cpp`, as `functional.cpp` is for `torch::nn::functional` tests.